packages/create-fuels/scripts/rewriteTemplatePackageJson.ts

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { versions } from '@fuel-ts/versions';
+import fs from 'fs/promises';
+import { join } from 'path';
+
+const replacePhraseWithSomething = async (phrase: RegExp, something: string) => {
+  const templatePackageJson = await fs.readFile(
+    join(__dirname, '../templates/nextjs/package.json'),
+    'utf-8'
+  );
+
+  const newPackageJson = templatePackageJson.replace(phrase, something);
+
+  await fs.writeFile(join(__dirname, '../templates/nextjs/package.json'), newPackageJson);
+};
+
+// This script injects the latest version of fuels into the package.json of the template, in place of workspace:*.
+const run = async () => {
+  await replacePhraseWithSomething(/"fuels": "workspace:\*"/, `"fuels": "${versions.FUELS}"`);
+
+  // replace "fuels:init" with "prebuild"
+  await replacePhraseWithSomething(/"fuels:init"/, `"prebuild"`);
+
+  // replace "build": "npm run fuels:init && next build", with "build": "next build"
+  await replacePhraseWithSomething(
+    /"build": "npm run fuels:init && next build"/,
+    `"build": "next build"`
+  );
+};
+
+run().catch(() => {
+  process.exitCode = 1;
+});
```

Comment:
Because it's just a script, I'd go with something very minimal.

```suggestion
import { versions } from '@fuel-ts/versions';
import { readFileSync, writeFileSync } from 'fs';
import { join } from 'path';

const filepath = join(__dirname, '../templates/nextjs/package.json');

let contents = readFileSync(filepath, 'utf-8');

contents = contents.replace(/xprebuild/g, 'prebuild');
contents = contents.replace(/"fuels": "workspace:\*"/, `"fuels": "${versions.FUELS}"`);
contents = contents.replace(/"build": "next build"/, `"build": "next build"`);

writeFileSync(filepath, contents);
```